source/docs/articles/sites/migrate/wordpress-site-networks.md

```diff
@@ -0,0 +1,136 @@
+3. Copy the SSH URL for the site repository, found in the [clone command](/docs/articles/local/starting-with-git/#step-2-copy-the-git-clone-command). **Do not copy `git clone` or the site name.** The URL should look similar to the following:
+
+ ```bash
+ ssh://codeserver.dev.{site-id}@codeserver.dev.{site-id}.drush.in:2222/~/repository.git
+ ```
+
+4. Use Git to pull in the upstream's code (which may have Pantheon-specific optimizations) to your existing site's codebase, replacing `<ssh_url>` with the SSH URL copied in step 3:
+
+ ```bash
+ git pull --no-rebase --squash -Xtheirs <ssh_url> master
+ ```  
+
+ Will yield:  
+ ```bash
+ Squash commit -- not updating HEAD  
+ Automatic merge went well; stopped before committing as requested
+ ```
+Preserve any logic necessary in the original `wp-config.php` and `.gitignore` files.
+5. Add Pantheon as a remote destination, replacing `<ssh_url>` with the SSH URL copied in Step 3:
+
+ ```bash
+ git remote add pantheon <ssh_url>
+ ```
+
+6. Run `git add` and `commit` to prepare the Pantheon core merge for pushing to the repository:
+ ```bash
+ git add -A
+ git commit -m "Adding Pantheon core files"
+ ```
+7. Push your newly merged codebase up to your Pantheon site repository:
+
+ ```bash
+ git push pantheon master --force
+ ```
+
+8. Go to the Code tab of your Dev environment on the Site Dashboard. The most recent commit adds Pantheon's core files. This process preserves the commit history for site's already utilizing version control and once pushed your pre-existing commits will be visible on the Dashboard.
+
+## Files
+
+**Files** - `/wp-content/uploads` is a symlink to the `~/files` directory. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc. For information on highly populated directories, see [Platform Considerations](/docs/articles/sites/platform-considerations/#highly-populated-directories).
+
+Files are stored separately from the site's code. Larger file structures can fail in the Dashboard import due to sheer volume. It's best to use a utility such as an SFTP client or rsync. The biggest issue is having the transfer stopped due to connectivity issues. To handle that scenario, try this handy bash script:  
+
+```bash
+ENV='ENV'
+SITE='SITEID'
+
+read -sp "Your Pantheon Password: " PASSWORD
+if [[ -z "$PASSWORD" ]]; then
+echo "Woops, need password"
+exit
+fi
+
+while [ 1 ]
+do
+sshpass -p "$PASSWORD" rsync --partial -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222'  $ENV.$SITE@appserver.$ENV.$SITE.drush.in:files/* --temp-dir=../tmp/  ./files/
+if [ "$?" = "0" ] ; then
+echo "rsync completed normally"
+exit
+else
+echo "Rsync failure. Backing off and retrying..."
+sleep 180
+fi
+done
+```
+This script connects to your Pantheon site's Dev environment and starts uploading your files. If an error occurs during transfer, rather than stopping, it waits 180 seconds and picks up where it left off.  

```

Comment:
Thanks for the tip on skipping the local download @danielbachhuber! I'm hoping to revise the [existing script](https://github.com/pantheon-systems/documentation/pull/1145/files#diff-c336aadafa7e4380adced72e591426a9R88) to demonstrate using rsync with 2 remote environments, do you have any tips or tricks for that since rsync doesn't allow the source and destination to both be remote?